src/components/pages/newReservation.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLocation } from 'react-router';
+import { createReservation } from '../../redux/reservation/thunk';
+import { fetchDoctors } from '../../redux/doctor/doctorSlice';
+import { selectStatus } from '../../redux/user/userSlice';
+import SideNav from '../navbar/SideNav';
+
+const NewReservation = () => {
```

Comment:
- [ ] I was able to reserve in the past:

![image](https://github.com/tajulafreen/Final-capstone-Frontend/assets/57421823/c5937320-37b5-4ccf-a855-6115cd5c7168)

Kindly, add some validation so your users can only reserve in the future. 👍